client/src/Components/Slider.tsx

```diff
@@ -0,0 +1,53 @@
+import React from "react";
```

Comment:
for the cmponents folder use sub folders like home,about etc 
and for the pages folder also  use sub folders

Reply:
done